packages/edit-site/src/components/block-editor/back-button.js

```diff
@@ -22,11 +23,12 @@ function BackButton() {
 	const history = useHistory();
 	const isTemplatePart = location.params.postType === TEMPLATE_PART_POST_TYPE;
 	const isNavigationMenu = location.params.postType === NAVIGATION_POST_TYPE;
+	const isPattern = location.params.postType === PATTERN_TYPES.user;
 	const previousTemplateId = location.state?.fromTemplateId;
 
```

Comment:
it would be nice to have some sort of `hasHistory` check in the site editor to decided if back button shown instead of these multiplying post type checks but I think we should do that as a follow-up PR.